AlphaWallet/Core/Types/RunLoopThread.swift

```diff
@@ -43,14 +43,14 @@ class RunLoopThread: Thread {
         }
     }
 
-    func performSync(_ block: @escaping () -> Swift.Void) {
```

Comment:
`performSync` - get called for all calls related to realm, when we fetch or write data to realm, all operations call on run loop thread. (maybe function name isn't good)